### PR TITLE
fix(bedrock_httpx.py): Add anthropic.claude-3-5-sonnet-20240620-v1:0 to converse list

### PR DESCRIPTION
**Pull request by Manouchehri**
Pretty trivial change.

### CODE REVIEW

**Comment by Manouchehri** [comment]
image

It's working as expected. :)

File: litellm/llms/bedrock_httpx.py [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "anthropic.claude-v2:1",
     "anthropic.claude-v1",
     "anthropic.claude-instant-v1",
+    "anthropic.claude-3-5-sonnet-20240620-v1:0",
```
Comment (ishaan-jaff) [inline litellm/llms/bedrock_httpx.py]: nit - can you move this to the top of this list

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
minor nit

File: litellm/llms/bedrock_httpx.py (hunk 1 above)
Comment (Manouchehri) [reply litellm/llms/bedrock_httpx.py]: Done.

**Review by ishaan-jaff — APPROVED** [review]
LGTM ! thanks @Manouchehri 😄

**Pull request merged**